src/behaviors/price_changer.rs

```diff
@@ -0,0 +1,117 @@
+use std::{fmt, sync::Arc};
+
+use anyhow::Result;
+use arbiter_core::middleware::ArbiterMiddleware;
+use arbiter_engine::messager::{Message, Messager};
+use ethers::types::H160;
+use RustQuant::{
+    models::*,
+    stochastics::{process::Trajectories, *},
+};
+
+use super::*;
+use crate::bindings::liquid_exchange::LiquidExchange;
+
+#[derive(Serialize, Deserialize)]
+pub struct PriceChanger {
+    #[serde(skip)]
+    pub params: OrnsteinUhlenbeckParams,
```

Comment:
In the future we can make this generic and have a `ProcessType` trait. Then impl that on the different param types.